x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/mock/stats/mock_stats_green_index.ts

```diff
@@ -14,280 +14,11 @@ import { IndicesStatsIndicesStats } from '@elastic/elasticsearch/lib/api/types';
  * where the index's `primaries.docs.count` and `total.docs.count` have
  * **different** values, `4` and `8`, per this mock `_stats` API output
  */
-export const mockStatsGreenIndex: Record<string, IndicesStatsIndicesStats> = {
+export const mockStatsGreenIndex: Record<string, MeteringStatsIndex> = {
   'auditbeat-custom-index-1': {
     uuid: 'jRlr6H_jSAysOLZ6KynoCQ',
-    health: 'green',
-    status: 'open',
```

Comment:
double check

Reply:
When health is `green` , its `primaries.docs.count` and `total.docs.count` have different values after deployed to the cloud. 

However, the `doc count` of both health `yellow` and `green` indices were all coming from mockStatsGreenIndex[indexName].primaries?.docs?.count. So when we are normalising the api formate, we should just use `primaries.docs.count` to retrieve the doc count.